ui/page/components/account_selector.go

```diff
@@ -297,6 +343,7 @@ func (asm *AccountSelectorModal) Handle() {
 	}
 
 	if asm.modal.BackdropClicked(asm.isCancelable) {
+		asm.onExit()
```

Comment:
`onExit` should also be called when the modal closes because an account was selected, yeah?